sample_install_only_tomcat_env.sh

```diff
@@ -0,0 +1,62 @@
+#!/usr/bin/env bash
+
+# The LabKey Install-Script is broken out into to step functions. Each STEP can be skipped by exporting the
```

Comment:
broken out into to step > broken out into step

Reply:
edited